src/content/configuration/other-options.md

```diff
@@ -167,9 +168,9 @@ module.exports = {
 
 ### `cache.store`
 
-`string: 'pack'`
+`string = 'pack': 'pack'`
```

Comment:
do i understand correctly that schema only allows `'pack'` or maybe also `false` or any other values?

Reply:
I checked just now, to be sure, and it looks like the only supported value (and hence, the default as well).
Also, that's what the docs explicitly say below in ```pack is the only supported mode since webpack 5.0.x```

Reply:
yeah looks like it. but best to be on the safe side and double check :) https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json#L264